contrib/add-capsule-header.py

```diff
@@ -45,27 +45,31 @@ def main(args):
             bin_data = bin_data[hdrsz_old:]
 
     # set header flags
-    flags = CAPSULE_FLAGS_PERSIST_ACROSS_RESET | CAPSULE_FLAGS_INITIATE_RESET
-    if args.flags:
-        flags = int(args.flags, 16)
+    flags = CAPSULE_FLAGS_PERSIST_ACROSS_RESET | CAPSULE_FLAGS_INITIATE_RESET | CAPSULE_FLAGS_POPULATE_SYSTEM_TABLE
```

Comment:
It looks like Windows unconditionally adds this too, in https://docs.microsoft.com/en-us/windows-hardware/drivers/bringup/processing-updates -- right?

Reply:
Well it does based on what was in the INF.  
* I think it's a better "default" to have it in at least (especially since it's definitely required for Dell system)
* add_capsule_header.py has an argument to override anyway.